assemblies/llama-stack-adv-examples.adoc

```diff
@@ -0,0 +1,12 @@
+:_module-type: ASSEMBLY
+
+ifdef::context[:parent-context: {context}]
+
+[id="llama-stack-adv-examples_{context}"]
+= Llama Stack application examples
+
+Llama Stack allows you to create custom applications in your {productname-short} cluster. 
```

Comment:
```suggestion
Llama Stack allows you to create AI-driven applications in your {productname-short} cluster. 
```

This sounds quite generice "create custom application", I would rephrase it to be a bit more specific. Maybe one could add some additional information to this intro that this chapter is all about Llama Stack examples ?